lib/VerifiableCredentialStore.js

```diff
@@ -38,25 +42,34 @@ export default class VerifiableCredentialStore {
    *
    * @param {string} [type]
    * @param {string} [issuer]
+   * @param {boolean} [displayable]
    *
    * @return {Promise<Array>} List of matching VCs
    */
-  async find({type, issuer}) {
-    const equals = [];
+  async find({type, issuer, displayable}) {
+    const equals = {};
     if(type) {
       if(Array.isArray(type)) {
-        const query = type.map(type => ({'content.type': type}));
-        equals.push(...query);
+        // needs to be implemented on the edv-client side first
+        throw new Error('"type" as array is not implemented.');
       } else {
-        equals.push({'content.type': type});
+        equals['content.type'] = type;
```

Comment:
I don't know anything about this, so can't speak to it, ignoring for this review.